boards/arm/efm32pg_stk3402a/doc/index.rst

```diff
@@ -189,6 +189,77 @@ terminal session:
 
    Hello World! arm
 
+Twister Testing on EFM32PG-STK3402A
+===================================
+
+A very simple test on the GPIO functionality can be found at:
+
+	``tests/drivers/gpio/gpio_basic_api``
+
+The Twister tests can be run as follows:
+
+.. code-block:: console
+
+   scripts/twister --device-testing --hardware-map <relative path to>-map.yml
+
+Documenting the EFM32PG-STK3402A
+********************************
+
+The Zephyr documentation is based upon
+`reStructuredText <http://sphinx-doc.org/rest.html>`__,
+and `Sphinx <http://sphinx-doc.org/>`__, which are text based tools
+that have limited facilities for `WYSIWYG (pronounced wiz-ee-wig) what you see is what you get:
+<https://www.techtarget.com/whatis/definition/WYSIWYG-what-you-see-is-what-you-get>`__.
+
+However, there are work arounds that can help you convert from
+various other tools to **rst** files. While this is not Pearl Gecko
+specific, it is included here, along with the changes above, as a
+convenience, until we push upstream. Then a more appropriate location
+can be found.
+
+I have a preference for
+`rich text editors <https://froala.com/blog/editor/a-beginners-guide-to-rich-text-editors/>`__\ ”
+and a promoter of
+`Literate Programming <https://en.wikipedia.org/wiki/Literate_programming>`__,
+but I’m using Microsoft’s Word to edit this text. I started from the
+original **rst** file and used `Pandoc <https://pandoc.org/>`__
+to convert to **docx**.
```

Comment:
what's going on? first, this doesn't seem to be related with the commit, and second, why do we need all this information in Zephyr docs?

Reply:
You don't I just hadn't found a better location for it, but I can remove it from here, at least.